src/main/java/org/jabref/gui/slr/ManageStudyDefinitionView.java

```diff
@@ -0,0 +1,326 @@
+
+    @FXML private TextField studyTitle;
+    @FXML private TextField addAuthor;
+    @FXML private TextField addResearchQuestion;
+    @FXML private TextField addQuery;
+    @FXML private ComboBox<StudyDatabase> databaseSelectorComboBox;
+    @FXML private TextField studyDirectory;
+
+    private final Button saveButton;
+
+    @FXML private Button addAuthorButton;
+    @FXML private Button addResearchQuestionButton;
+    @FXML private Button addQueryButton;
+    @FXML private Button addDatabaseButton;
+    @FXML private Button selectStudyDirectory;
+    @FXML private ButtonType saveButtonType;
+    @FXML private Label helpIcon;
+
+    @FXML private ListView<String> authorListView;
+    @FXML private ListView<String> questionListView;
+    @FXML private ListView<String> queryListView;
+    @FXML private TableView<StudyDatabase> databaseTableView;
+
+    @FXML private TableColumn<StudyDatabase, String> databaseTableEntry;
+    @FXML private TableColumn<StudyDatabase, StudyDatabase> enabledTableEntry;
+    @FXML private TableColumn<StudyDatabase, StudyDatabase> removeTableEntry;
+
+    /**
+     * This can be used to either create new study objects or edit existing ones.
+     *
+     * @param study          null if a new study is created. Otherwise the study object to edit.
+     * @param studyDirectory the directory where the study to edit is located (null if a new study is created)
+     */
+    public ManageStudyDefinitionView(Study study, Path studyDirectory, ImportFormatPreferences importFormatPreferences, DialogService dialogService, Path workingDirectory) {
+        // If an existing study is edited, open the directory dialog at the directory the study is stored
+        this.workingDirectory = Objects.isNull(studyDirectory) ? workingDirectory : studyDirectory;
+        this.setTitle(Objects.isNull(studyDirectory) ? Localization.lang("Define study parameters") : Localization.lang("Manage study definition"));
+        this.dialogService = dialogService;
+        viewModel = new ManageStudyDefinitionViewModel(study, studyDirectory, importFormatPreferences);
+        ViewLoader.view(this).load().setAsDialogPane(this);
+
+        saveButton = ((Button) this.getDialogPane().lookupButton(saveButtonType));
+
+        setResultConverter(button -> {
+            if (button == saveButtonType) {
+                return viewModel.saveStudy();
+            }
+            // Cancel button will return null
+            return null;
+        });
+        registerListenersAndBindings();
+    }
+
+    @FXML
+    private void initialize() {
+        setButtonIcons();
+        setButtonToolTips();
+        setKeyPressListenersForInputFields();
+        setDataForListViews();
+        setCellFactories();
+        configureLayout();
+    }
+
+    /**
+     * Configures all the buttons used directly in the study management GUI
+     */
+    private void setButtonIcons() {
+        addAuthorButton.setGraphic(IconTheme.JabRefIcons.ADD_ARTICLE.getGraphicNode());
```

Comment:
These can be set in the fxml as well (via a control IconView or GryphView or somethling like this)